src/by/incubator/Main.java

```diff
@@ -0,0 +1,4 @@
+package by.incubator;
+
+public class Main {
```

Comment:
No way, it is needed so that the project structure is by/incubator/ . After creating another task 2 folder, everything will become normal and you can remove this class